src/webview/markdownEngine.ts

```diff
@@ -107,4 +138,29 @@ class MarkdownEngine {
     }
 }
 
+// tslint:disable-next-line: max-classes-per-file
+class MarkdownConfiguration {
+
+    public readonly extRoot: string; // root path of vscode built-in markdown extension
+    public readonly lineHeight: number;
+    public readonly fontSize: number;
+    public readonly fontFamily: string | undefined;
+
+    public constructor() {
+        const markdownConfig: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration("markdown");
+        this.extRoot = path.join(vscode.env.appRoot, "extensions", "markdown-language-features");
+        this.lineHeight = Math.max(0.6, +markdownConfig.get<number>("preview.lineHeight", NaN));
+        this.fontSize = Math.max(8, +markdownConfig.get<number>("preview.fontSize", NaN));
+        this.fontFamily = this.resolveFontFamily(markdownConfig);
+    }
+
+    private resolveFontFamily(config: vscode.WorkspaceConfiguration): string | undefined {
+        let fontFamily: string | undefined = config.get<string | undefined>("preview.fontFamily", undefined);
+        if (isWindows() && fontFamily && fontFamily === config.inspect<string>("preview.fontFamily")!.defaultValue) {
+            fontFamily = `${fontFamily}, 'Microsoft Yahei UI'`;
```

Comment:
VS Code in Windows uses SongTi to render Chinese characters. So, if a user didn't change the default `preview.fontFamily` settings, we prepend a YaHei to it to represent Chinese texts.

As for the licensing of the font, please refer to following links:
* https://stackoverflow.com/questions/1805324/how-to-use-web-fonts-legally
* https://www.zhihu.com/question/314779554

Generally speaking, it won't be illegal if we only _reference_ the font, in comparation with _including_ the font by things like `font-face`.